src/site/content/en/mini-apps/what-are-super-apps/index.md

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
`web.dev/mini-apps-super-apps`?

`web.dev/super-apps-introduction`?

`web.dev/super-apps`?

(Don't like the last two as much because we lose our `mini-app` prefix

Reply:
Done.